src/lightwingFaktoryWorker/templates/deployment.yaml

```diff
@@ -80,6 +80,23 @@ spec:
               value: {{ .Values.global.ccm.gcpProjectId }}
             - name: LIGHTWING_CLOUD-CONNECTOR_REMOTE_API_TOKEN_SECRET
               value: IC04LYMBf1lDP5oeY4hupxd4HJhLmN6azUku3xEbeE3SUx5G3ZYzhbiwVtK4i7AmqyU9OZkwB4v8E9qM
+            {{- if .Values.clickhouse.enabled }}
+            - name: LIGHTWING_CLICKHOUSE_ENABLED
+              value: {{ .Values.clickhouse.enabled | quote}}
```

Comment:
I see we are reusing certain secrets over and over, will it be better to create a secret.yaml and reference that entity?

Reply:
I have moved all the env variables into a config map